websites/G/Gartic.io/metadata.json

```
@@ -4,13 +4,19 @@
 		"name": "Striker",
 		"id": "215509157837537280"
 	},
+	"contributors": [
+		{
+			"name": "0×4a69726b61",
+			"id": "238728915647070209"
+		}
+	],
 	"service": "Gartic.io",
 	"description": {
 		"en": "Play Gartic.io online and for free. Draw and guess the words as fastest as you can! Join on worldwide rooms or create your own.",
 		"nl": "Speel Gartic.io online en gratis. Teken en raad de woorden zo snel als je kunt! Doe mee aan wereldwijde rooms of maak je eigen."
 	},
 	"url": "gartic.io",
-	"version": "1.3.17",
+	"version": "1.4.0",
 	"logo": "https://i.imgur.com/JkOc5Vo.png",
```

Now that the presence is edited, lets change the logo to 512x512
```suggestion
	"logo": "https://i.imgur.com/3rfHXtU.png",
```